core/kazoo_media/src/kz_media_continuous_proxy.erl

```diff
@@ -64,7 +70,7 @@ handle(Req0, {Meta, Bin}) ->
     lager:debug("media: ~s content-type: ~s size: ~b", [MediaName, ContentType, Size]),
 
     Req2 = case ContentType of
-               CT when CT =:= <<"audio/mpeg">> orelse CT =:= <<"audio/mp3">> ->
+               CT when CT =:= <<"audio/mpeg">>; CT =:= <<"audio/mp3">> ->
```

Comment:
The build job complained because the formatter wants `orelse` and `andalso` dropped to new lines. Replacing `orelse` with `;` changes the behaviour of the code (both conditions are evaluated vs the `orelse` which short-circuits when a 'true' condition is found). Also, we prefer `andalso` and `orelse` as they're more obvious that `,` and `;`, particularly with guards. So could you revert this change and drop the `orelse` condition down a line?